### ISSUE TITLE
[🐛 Bug]: file upload did not work anymore since firefox 117

### ISSUE THREAD

**Issue by gimler** [SeleniumHQ/selenium#12780]
### What happened?

Simple file upload did not work with firefox 117 image.

We did not touch our code only the container image. It works with chrome and firefox <117.

But did not work with
standalone-firefox:117.0-geckodriver-0.33-grid-4.12.1-20230912

When i mount the directory into the container it works. I found this issue which seams similar https://github.com/SeleniumHQ/docker-selenium/issues/1324

### How can we reproduce the issue?

```shell
create simple form and try to upload a file

<form method="post" enctype="multipart/form-data">
<input type="file" name="file" id="file">
</form>
```

### Relevant log output

```shell
faliure standalone-firefox:117.0-geckodriver-0.33-grid-4.12.1-202312

firefox-1  | 13:24:19.672 INFO [LocalDistributor.newSession] - Session request received by the Distributor: 
firefox-1  |  [Capabilities {acceptInsecureCerts: true, browserName: firefox}]
firefox-1  | 13:24:19.765 WARN [SeleniumManager.lambda$runCommand$1] - Exception managing firefox: Unable to discover proper geckodriver version in offline mode
firefox-1  | 13:24:21.219 INFO [LocalNode.newSession] - Session created by the Node. Id: 663088f3-6c16-4811-80ff-0cdff9c700a2, Caps: Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 117.0, moz:accessibilityChecks: false, moz:buildID: 20230824132758, moz:geckodriverVersion: 0.33.0, moz:headless: false, moz:platformVersion: 6.2.0-32-generic, moz:processID: 165, moz:profile: /tmp/rust_mozprofile8BWzMd, moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platformName: linux, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://172.23.0.9:5555/sessio..., se:noVncPort: 7900, se:vnc: ws://172.23.0.9:5555/sessio..., se:vncEnabled: true, se:vncLocalAddress: ws://172.23.0.9:7900, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
firefox-1  | 13:24:21.222 INFO [LocalDistributor.newSession] - Session created by the Distributor. Id: 663088f3-6c16-4811-80ff-0cdff9c700a2 
firefox-1  |  Caps: Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 117.0, moz:accessibilityChecks: false, moz:buildID: 20230824132758, moz:geckodriverVersion: 0.33.0, moz:headless: false, moz:platformVersion: 6.2.0-32-generic, moz:processID: 165, moz:profile: /tmp/rust_mozprofile8BWzMd, moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platformName: linux, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://172.23.0.9:5555/sessio..., se:noVncPort: 7900, se:vnc: ws://172.23.0.9:5555/sessio..., se:vncEnabled: true, se:vncLocalAddress: ws://172.23.0.9:7900, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
firefox-1  | 13:24:21.666 WARN [SeleniumSpanExporter$1.lambda$export$3] - {"traceId": "93b4ac2d9a302701623e11ffa848968f","eventTime": 1695129861664298003,"eventName": "HTTP request execution complete","attributes": {"http.flavor": 1,"http.handler_class": "org.openqa.selenium.remote.http.Route$PredicatedRoute","http.host": "firefox:5555","http.method": "POST","http.request_content_length": "3384","http.scheme": "HTTP","http.status_code": 405,"http.target": "\u002fsession\u002f663088f3-6c16-4811-80ff-0cdff9c700a2\u002ffile"}}
firefox-1  | 
firefox-1  | 13:24:21.675 WARN [SeleniumSpanExporter$1.lambda$export$3] - {"traceId": "e0de4216a7d8ce67d58d045552878cc4","eventTime": 1695129861674459995,"eventName": "HTTP request execution complete","attributes": {"http.flavor": 1,"http.handler_class": "org.openqa.selenium.remote.http.Route$PredicatedRoute","http.host": "firefox:5555","http.method": "POST","http.request_content_length": "93","http.scheme": "HTTP","http.status_code": 400,"http.target": "\u002fsession\u002f663088f3-6c16-4811-80ff-0cdff9c700a2\u002felement\u002f5ba700f6-e434-4b51-aefa-c7e05318fa6b\u002fvalue"}}
firefox-1  | 
firefox-1  | 13:24:23.335 INFO [LocalSessionMap.lambda$new$0] - Deleted session from local Session Map, Id: 663088f3-6c16-4811-80ff-0cdff9c700a2
firefox-1  | 13:24:23.336 INFO [GridModel.release] - Releasing slot for session id 663088f3-6c16-4811-80ff-0cdff9c700a2
firefox-1  | 13:24:23.336 INFO [SessionSlot.stop] - Stopping session 663088f3-6c16-4811-80ff-0cdff9c700a2
```

success standalone-firefox:116.0-geckodriver-0.33-grid-4.11.0-20230801
```
firefox-1  | 13:23:33.074 INFO [LocalDistributor.newSession] - Session request received by the Distributor: 
firefox-1  |  [Capabilities {acceptInsecureCerts: true, browserName: firefox}]
firefox-1  | 13:23:34.862 INFO [LocalNode.newSession] - Session created by the Node. Id: 020dde8e-4864-4b85-a0a9-f004a5cd5f77, Caps: Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 116.0, moz:accessibilityChecks: false, moz:buildID: 20230727152340, moz:geckodriverVersion: 0.33.0, moz:headless: false, moz:platformVersion: 6.2.0-32-generic, moz:processID: 175, moz:profile: /tmp/rust_mozprofilecczzUn, moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platformName: linux, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://172.23.0.9:5555/sessio..., se:noVncPort: 7900, se:vnc: ws://172.23.0.9:5555/sessio..., se:vncEnabled: true, se:vncLocalAddress: ws://172.23.0.9:7900, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
firefox-1  | 13:23:34.866 INFO [LocalDistributor.newSession] - Session created by the Distributor. Id: 020dde8e-4864-4b85-a0a9-f004a5cd5f77 
firefox-1  |  Caps: Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 116.0, moz:accessibilityChecks: false, moz:buildID: 20230727152340, moz:geckodriverVersion: 0.33.0, moz:headless: false, moz:platformVersion: 6.2.0-32-generic, moz:processID: 175, moz:profile: /tmp/rust_mozprofilecczzUn, moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platformName: linux, proxy: Proxy(), se:bidiEnabled: false, se:cdp: ws://172.23.0.9:5555/sessio..., se:noVncPort: 7900, se:vnc: ws://172.23.0.9:5555/sessio..., se:vncEnabled: true, se:vncLocalAddress: ws://172.23.0.9:7900, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
firefox-1  | 13:23:37.001 INFO [LocalSessionMap.lambda$new$0] - Deleted session from local Session Map, Id: 020dde8e-4864-4b85-a0a9-f004a5cd5f77
firefox-1  | 13:23:37.002 INFO [GridModel.release] - Releasing slot for session id 020dde8e-4864-4b85-a0a9-f004a5cd5f77
firefox-1  | 13:23:37.003 INFO [SessionSlot.stop] - Stopping session 020dde8e-4864-4b85-a0a9-f004a5cd5f77
```

### Operating System

Ubuntu

### Selenium version

Webdriver

### What are the browser(s) and version(s) where you see this issue?

Firefox 117

### What are the browser driver(s) and version(s) where you see this issue?

geckodriver-0.33

### Are you using Selenium Grid?

4.12.1

**Comment by titusfortner** [comment]
Can you turn on client side logging to see what is getting sent to the grid?

https://www.selenium.dev/documentation/webdriver/troubleshooting/logging/

**Comment by rmorris227** [comment]
I left a similar report on StackOverflow - It contains an additional detail from the Dev console.
[https://stackoverflow.com/questions/77138195/code-injection-technique-with-selenium-for-file-uploads-stopped-working-with-chr](url)

**Comment by titusfortner** [comment]
Can you check if it works in Selenium 4.11?

**Comment by gimler** [comment]
there is no docker container image with firefox 117 and selenium 4.11 only 4.12 ;(

**Comment by titusfortner** [comment]
Well, I think this us related to a bug in 4.12 so I guess wait for 4.13 and we can see

**Comment by rmorris227** [comment]
Updates to Chrome and Selenium (92, 9200) did not solve the problem I am seeing.

**Comment by rmorris227** [comment]
The problem I was seeing was fixed with the update to 4.13.1

**Comment by gimler** [comment]
the last image from yesterday has the new firefox but not the new grid version 4.13.1 ;(
https://hub.docker.com/r/selenium/standalone-firefox

@titusfortner where is this configured?
